hybrid_content_manager/ipycompat.py

```diff
@@ -0,0 +1,19 @@
+"""Utilities for managing compat between notebook versions."""
+
+#  if notebook.version_info[0] >= 6:  # noqa
+#  raise ImportError("Jupyter Notebook versions 6 and up are not supported.")
```

Comment:
Delete these comments now that we removed the `pgcontents` code that was incompatible with `notebook >= 6.0.0`